Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.7
+
+#Run python in unbuffered mode to allow for log messages to be
+#immediately dumped to the stream instead of being buffered.
+ENV PYTHONUNBUFFERED 1
+
+RUN apt-get update
+
+# Copying app to docker and making it as working directory
+RUN mkdir /app
+WORKDIR /app
+COPY ./ /app
+
+# Install all dependencies
+RUN pip install pipenv
+RUN pipenv install --system
+
```

Comment:
Please update pip before installing python dependencies and add `--dev` flag to `pipenv install` command. You can check this [Dockerfile](https://github.com/creativecommons/cccatalog-api/blob/master/cccatalog-api/Dockerfile#L14:L21) for reference.
In my view, we don't need to copy the whole app code if we use docker volumes in the docker-compose.

Reply:
Right @krysal, In the development environment, we won't need an independent docker image hence no need to copy whole app.